modul/player.py

```
@@ -1,39 +1,66 @@
+        """TODO: add docstring."""
+        super().__init__(x, y, C.PLAYER_RADIUS)
         self.rotation = 0
         self.velocity = pygame.Vector2(0, 0)
         self.shoot_timer = 0
         self.invincible = False
         self.invincible_timer = 0
         self.sounds = Sounds()
         self.shield_active = False
+        try:
+            from modul import input_utils
+        except Exception:  # pragma: no cover - minimal stub for runtime when unavailable
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)